setup.md

```diff
@@ -121,108 +121,35 @@ $ git config --global core.editor "nano -w"
-via the `-C` parameter as shown below.
-Note that the `ssh-keygen` command can be run with different parameters -
-e.g. to select a specific public key algorithm and key length;
-if you do not use them `ssh-keygen` will generate an
-[RSA](https://en.wikipedia.org/wiki/RSA_(cryptosystem)#:~:text=RSA%20involves%20a%20public%20key,by%20using%20the%20private%20key.)
-key pair for you by default.
-GitHub now recommends that you use a newer cryptographic standard (such as [EdDSA](https://en.wikipedia.org/wiki/EdDSA) variant algorithm [Ed25519](https://cryptobook.nakov.com/digital-signatures/eddsa-and-ed25519)),
-so please be sure to specify it using the `-t` flag as shown below.
-It will also prompt you to answer a few questions -
-e.g. where to save the keys on your machine and a passphrase to use to protect your private key.
-Pressing 'Enter' on these prompts will get `ssh-keygen` to use the default key location (within
-`.ssh` folder in your home directory)
-and set the passphrase to empty.
+one public that you upload to your GitLab account, and one private that remains on your machine.
 
-~~~
-$ ssh-keygen -t ed25519 -C "your-github-email@example.com"
-~~~
-{: .language-bash}
-~~~
-Generating public/private ed25519 key pair.
-Enter file in which to save the key (/Users/<YOUR_USERNAME>/.ssh/id_ed25519): 
-Enter passphrase (empty for no passphrase): 
-Enter same passphrase again: 
-Your identification has been saved in /Users/<YOUR_USERNAME>/.ssh/id_ed25519
-Your public key has been saved in /Users/<YOUR_USERNAME>/.ssh/id_ed25519.pub
-The key fingerprint is:
-SHA256:qjhN/iO42nnYmlpink2UTzaJpP8084yx6L2iQkVKdHk your-github-email@example.com
-The key's randomart image is:
-+--[ED25519 256]--+
-|.. ..            |
-| ..o A           |
-|. o..            |
-| .o.o .          |
-| ..+ =  B        |
-| .o = ..         |
-|o..X *.          |
-|++B=@.X          |
-|+*XOoOo+         |
-+----[SHA256]-----+
-~~~
-{: .output}
+GitLab provides full documentation and guides on how to on [this page](https://docs.gitlab.com/ee/user/ssh.html).
+Please follow the instructions to setup an SSH connection.
 
-Next, you need to copy your public key (**not your private key - this is important!**) over to 
-your GitHub account. The `ssh-keygen` command above will let you know where your public key is saved (the file should have the 
-extension ".pub"), and you can get its contents (e.g. on a Mac OS system) as follows:
-~~~
-$ cat /Users/<YOUR_USERNAME>/.ssh/id_ed25519.pub
-~~~
-{: .language-bash}
-~~~
-ssh-ed25519 AABAC3NzaC1lZDI1NTE5AAAAICWGVRsl/pZsxx85QHLwSgJWyfMB1L8RCkEvYNkP4mZC your-github-email@example.com
-~~~
-{: .output}
-
-Copy the line of output that starts with "ssh-ed25519" and ends with your email address 
-(it may start with a different algorithm name based on which one you used to generate the key pair 
-and it may have gone over multiple lines if your command line window is not wide enough).
+To check that everything works, run:L
```

Comment:
I believe the trailing `L` should not be there.
```suggest
To check that everything works, run:
```

Reply:
Check, cowboy fixed that in the main branch!